cirq-core/cirq/ops/moment.py

```diff
@@ -49,6 +49,18 @@ def _default_breakdown(qid: 'cirq.Qid') -> Tuple[Any, Any]:
         return None, qid
 
 
+def _have_equal_gates(op1, op2):
+    """Compares two operations and returns a boolean of it their gates
+    match.
+    Returns:
+        bool: True if both gates are not null, the same, and take in
+           the same number of qubits from their operations.
+    """
+    if op1.gate is None or op2.gate is None:
+        return False
+    return len(op1.qubits) == len(op2.qubits) and op1.gate == op2.gate
```

Comment:
Why does the number of qubits matter?  If I have cirq.measure(a), cirq.measure(b), cirq.measure(c) that returns MesaurementGate, but cirq.measure(a,b), cirq.measure(c) returns None?  That's unexpected.

Reply:
Done.